metabase/metabase
# Add properties to query_execution

Closes https://github.com/metabase/metabase/issues/35176

### Description

Adding a function to check whether a query will have sandboxing in the future. I did try to swap to ordering of query processing in userland such that we wouldn't have to run some of the sandboxing code twice, but that caused a lot of unexpected side effects, so I reverted to the way I have now.

### How to verify

1. Create a sandboxed user on a certain table and query that table
2. Check the Query execution table to see is_sandboxed = true
